docs/notebooks/measurement-error-mitigation.ipynb

```diff
@@ -0,0 +1,546 @@
+    "    calibration = result.json()\n",
+    "    return calibration"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
+    "\n",
+    "We will measure a Bell state, first without error mitigation, and then with error mitigation. We perform the calculations on real hardware (or on a simulator when the hardware is not available)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def spin_2_noise_model(p: float = 0.01):\n",
+    "    \"\"\" Define noise model representation for Spin-2 \"\"\"\n",
+    "    noise_model = NoiseModel()\n",
+    "    error_gate = pauli_error([(\"X\", p), (\"I\", 1 - p)])\n",
+    "\n",
+    "    noise_model.add_all_qubit_quantum_error(error_gate, [\"u1\", \"u2\", \"u3\", \"rx\", \"ry\", \"x\", \"y\"])\n",
+    "\n",
+    "    read_err = ReadoutError([[0.9, 0.1], [0.2, 0.8]])\n",
+    "    noise_model.add_readout_error(read_err, [0])\n",
+    "    read_err = ReadoutError([[0.7, 0.3], [0.25, 0.75]])\n",
+    "    noise_model.add_readout_error(read_err, [1])\n",
+    "    return noise_model\n",
+    "\n",
+    "\n",
+    "noise_model_spin2 = spin_2_noise_model()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Set up connection to QI"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We expect the user to have set up the token authentication as described [here](https://github.com/qutech-delft/quantuminspire#configure-your-token-credentials-for-quantum-inspire) in order to be able to access the Spin-2 backend in the following cells."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Enter email:\n",
+      "example@email.com\n",
+      "Enter password\n",
+      "········\n",
+      "QI connection not available, fall-back to simulation.\n"
+     ]
+    }
+   ],
+   "source": [
+    "try:\n",
+    "    authentication = get_authentication()\n",
+    "    qi_api = QuantumInspireAPI(authentication=authentication)\n",
+    "    QI.set_authentication(authentication)\n",
+    "    qi_backend = QI.get_backend(\"Spin-2\")\n",
+    "except:\n",
+    "    print(\"QI connection not available, fall-back to simulation.\")\n",
+    "    qi_api = None\n",
+    "    qi_backend = None"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def execute_circuit_spin2(qc, qi_backend, shots=2048):\n",
+    "    \"\"\"  Execute circuit on Spin-2 with fall-back to simulation \"\"\"\n",
+    "    try:\n",
+    "        logging.info(f\"execute_circuit_spin2: run on {qi_backend}\")\n",
+    "        qi_job = execute(qc, backend=qi_backend, shots=shots)\n",
+    "\n",
+    "        result = qi_job.result()\n",
```

Comment:
When Spin-2 is OFFLINE, this may take quit a while, waiting for Spin-2 to come ONLINE and pick up the job.
When you want to check if Spin-2 is OFFLINE, you can do:
```
        api = QI.get_api()
        backend = api.get_backend_type(qi_backend.backend_name)
        if backend['status'] == 'OFFLINE':
            raise Exception("Backend is OFFLINE")
```